medicines/web/src/components/cookie-form/index.tsx

```diff
@@ -57,6 +57,8 @@ const StyledContentWrapper = styled.main`
   }
 `;
 
+const domain = '.windows.net';
```

Comment:
Should be `.mhra.gov.au`, or derived from `window.location`.

Reply:
The domain of the GA cookies specifically is `.windows.net` and require this specifically to be cleared, so you can't auto extract this from the window location and you also can't directly access the cookie domain (or other metadata) from JS, I believe. This could be moved into an environment variable, which I see as only being necessary if the site was going to be deployed on a different platform.

Reply:
(lol @ me for the `.au` above; old habits die hard)

The domain of the cookies in production is `.mhra.gov.uk`. As far as I understand it, attempting to set a cookie on a different domain just won't work.

<img width="1402" alt="Screenshot 2020-03-16 at 11 48 40" src="https://user-images.githubusercontent.com/76330/76755652-21450b00-677c-11ea-8503-54757f64b032.png">

Reply:
Actually, I was looking at the staging site, so ignore this. I'll have a think about the best way to do this.